templates/shared/_card--masterclass.html

```diff
@@ -8,9 +8,11 @@
     {% endif %}
   </a>
   <div class="p-card__inner">
-    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
+    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a>
+    </h3>
     <p>{{ session.Description }}</p>
+    <p class="u-text--muted u-no-margin--bottom">{{ session.Tags }}</p>
+    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
```

Comment:
Shall we remove the duration? It is not accurate and probably not so important when browsing the videos, wdyt?

Reply:
Or keep it but remove the "duration:" prefix

Reply:
I think it’s good to have Duration there. In future we can add a filter by duration as well. And yes we can get rid of “Duration: “ prefix for sure!